### ISSUE TITLE
Ract is not defined

### ISSUE THREAD

**Issue by eliezedeck** [parhelium/package-templating-ractive#1]
I am getting this error while using this package:

```
Exception from Tracker afterFlush function:
debug.js:41 ReferenceError: Ract is not defined
    at Template.voting.rendered (voting.js:4)
    at template.js:116
    at Function.Template._withTemplateInstanceFunc (template.js:457)
    at fireCallbacks (template.js:112)
    at null.<anonymous> (template.js:205)
    at view.js:107
    at Object.Blaze._withCurrentView (view.js:538)
    at view.js:106
    at Object.Tracker._runFlush (tracker.js:497)
    at onGlobalMessage (setimmediate.js:102)
```

Using latest Meteor version (1.2.1 at the time of writing).

Blaze is somewhat a bit too limited to what you can do, so Ractive is a much better alternative. Combining the good from current Meteor with Ractive is I think a terrific combination.

**Comment by parhelium** [comment]
Check `.meteor/packages` if you have entries:

```
parhelium:ractive
parhelium:templating-ractive
```

Few minutes ago I have update example of [ractive integration](https://github.com/parhelium/laboratory/tree/master/meteor/ractive-integration) to meteor@1.2.1 and it works.

**Comment by eliezedeck** [comment]
Yes, it turned-out that I didn't know I needed the pahelium:ractive
package, I though the templating-ractive pulled the necessary dependencies
to work.

So, I figured that out on my own. I guess this can be closed now.

But, if it is possible to pull the dependency, that would be great as I'm
sure others might stumble on this, and may have already tried and just
decided this things doesn't even work. But I confirm it does work.

Best regards,
Elie Zedeck

**Comment by parhelium** [comment]
`parhelium:templating-ractive@0.7.3_1` was released which implies `parhelium:ractive@0.7.3`. You don't need any more to include `parhelium:ractive`.

Thanks !
